File: app/src/main/java/com/antyzero/smoksmog/ui/typeface/TypefaceProvider.java
package com.antyzero.smoksmog.ui.typeface;


import android.content.Context;
import android.graphics.Typeface;

public class TypefaceProvider {

    private final Typeface defaultTypeface;

    public TypefaceProvider( Context context ) {
        defaultTypeface = Typeface.createFromAsset( context.getAssets(), "fonts/Lato-Light.ttf" );
    }

    public Typeface getDefault() {
        return defaultTypeface;
    }
}

File: app/src/main/kotlin/com/antyzero/smoksmog/ui/typeface/TypefaceProvider.kt
package com.antyzero.smoksmog.ui.typeface


import android.content.Context
import android.graphics.Typeface

class TypefaceProvider(context: Context) {

    val default: Typeface

    init {
        default = Typeface.createFromAsset(context.assets, "fonts/Lato-Light.ttf")
    }
}
